web/ui/src/Pools/Pool.js

```diff
@@ -0,0 +1,80 @@
+(function () {
+    'use strict';
+
+    // share angular services outside of angular context
+    let POOL_PERMISSIONS, $q, resourcesFactory;
+
+    controlplane.factory('Pool', PoolFactory);
+
+    // TODO - this is a dummy class till the fo'real one is in
+    // TODO - import real Host
+    function Host(model){
```

Comment:
This is the minimal type needed to get a working Host on the Pools page. This will be replaced with the full Host type in the upcoming commits for the 101 pools UI changes.